site/content/docs/5.0/utilities/spacing.md

```diff
@@ -93,3 +95,17 @@ The syntax is nearly the same as the default, positive margin utilities, but wit
   margin-top: -0.25rem !important;
 }
 ```
+
+## Gap
```

Comment:
Maybe we should move Negative margin and Horizontal centering to an `h2`?

Reply:
Opted for another organization to separate margin and padding from gap, since the two behave differently.